assets/install-scripts/install-teleport-tests/run-test.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# $1: edition: cloud, oss, or enterprise
+function test_teleport() {
+    if ! type teleport; then
+      echo "INSTALL_SCRIPT_TEST_FAILURE: teleport not found"
+    fi
+}
+
+function test_tctl() {
+    if ! type tctl; then
+    	echo "INSTALL_SCRIPT_TEST_FAILURE: tctl not found"
+    fi
+}
+
+function test_tsh() {
+    if ! type tsh; then
+    	echo "INSTALL_SCRIPT_TEST_FAILURE: tsh not found"
+    fi
+}
+
+function test_tbot() {
+    if ! type tctl; then
+    	echo "INSTALL_SCRIPT_TEST_FAILURE: tbot not found"
+    fi
+}
+
+function test_upgrader() {
+    if ! type teleport-upgrade; then
+    	echo "INSTALL_SCRIPT_TEST_FAILURE: upgrader not found"
+    fi
+}
+
+echo "RUNNING TEST"
+
+case $1 in
+    cloud)
+    	test_teleport
+    	test_tctl
+    	test_tsh
+    	test_tbot
+    	test_upgrader
+    ;;
+    oss)
+    	test_teleport
+    	test_tctl
+    	test_tsh
+    	test_tbot
+    ;;
+    enterprise)
+    	test_teleport
+    	test_tctl
+    	test_tsh
+    	test_tbot
```

Comment:
should we also test that `fdpass-teleport` is installed? That's probably a later version than 15.0.0 but I note the install script does mention it if it exists.

Although now I think on it more, I think it might be better to remove the tests for `tctl`, `tsh` and `tbot` - the installed does nothing specific to install those - it just installs a teleport release asset so we should only be checking that it has done that, not that the release asset contains the correct binaries - that test belongs somewhere else in the release process perhaps. Testing for the upgrader for cloud is correct as the install script has special logic for that.

Reply:
Ah, good point. Changed.